src/Facets/CalldataVerificationFacet.sol

```diff
@@ -111,16 +110,7 @@ contract CalldataVerificationFacet {
     function extractNonEVMAddress(
         bytes calldata data
     ) external pure returns (bytes32 nonEVMAddress) {
-        bytes memory callData;
-
-        if (
-            bytes4(data[:4]) == StandardizedCallFacet.standardizedCall.selector
-        ) {
-            // standardizedCall
-            callData = abi.decode(data[4:], (bytes));
-        } else {
-            callData = data;
-        }
+        bytes memory callData = data;
```

Comment:
I think we could use `data` directly below without defining new `callData` variable

Reply:
nope
![image](https://github.com/user-attachments/assets/ec13c233-34f4-4a2c-aff5-23dffb197d11)